src/mapblock_mesh.cpp

```diff
@@ -1064,14 +1067,14 @@ MapBlockMesh::MapBlockMesh(MeshMakeData *data, v3s16 camera_offset):
 		Convert FastFaces to MeshCollector
 	*/
 
-	MeshCollector collector;
+	MeshCollector collector(m_use_tangent_vertices);
 
 	{
 		// avg 0ms (100ms spikes when loading textures the first time)
 		// (NOTE: probably outdated)
 		//TimeTaker timer2("MeshCollector building");
 
-		for(u32 i=0; i<fastfaces_new.size(); i++)
+		for(u32 i = 0; i < fastfaces_new.size(); i++)
```

Comment:
if you already fix the style, then put the { brace on the same line, and add a space to get `for (`.